test/helpers/api-unit.helper.js

```diff
@@ -64,3 +65,32 @@ export function generateHistory (days) {
 
   return history;
 }
+
+export async function generateHabit (update = {}) {
```

Comment:
Since these are unit tests you don't need to save the objects to the database and subsequentially to use async functions. Just remove all the await task.save() and convert the function to normal sync functions